app/controllers/user_registrations_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UserRegistrationsController < ApplicationController
+  def edit
+    @token = params[:id]
+    @user = User.find_by(registration_token: params[:id])
+
+    if @user.blank?
+      not_authenticated
+      return
+    end
+
+  end
+
+  def update
+
+    @token = params[:id]
+    @user = User.find_by(registration_token: params[:id])
+
+    if @user.blank?
+      not_authenticated
+      return
+    end
+    if @user.update_attributes(user_params)
+      login(@user.email, user_params[:password])
+      redirect_to(dashboard_mentor_profiles_path, :notice => "You've been succesfully added as Mentor.")
```

Comment:
I have added automatically login for mentor, anjali will further add condition for mentee. I am closing both my task..Registration and automatic login to dashboard :)